content/en/profiler/enabling/ssi.md

```diff
@@ -3,7 +3,7 @@ title: Single Step Instrumentation for Profiling
 private: true
 ---
 
-<div class="alert alert-info">Enabling Profiling using Single Step Instrumentation is in private beta.</div>
```

Comment:
"Preview" feels right for SSI. But Attila pointed out that there could be concern about charging for things in preview releases. Definitions are a little confusing here, but I think we're OK with the change as proposed.